### DISCUSSION TITLE
Dynamite Core Team Meeting June 14th

### DISCUSSION THREAD

**Discussion by boeckera** [dynamics-of-stellar-systems/dynamite#390] [Announcements]
Dear @dynamics-of-stellar-systems/dynamite-maintainers,

the next dynamite meeting will take place on June 14th. Due to holidays this Thursday, we will skip the meeting on Friday as many people are away.

Since we all have time until the next meeting please think about what you would like to discuss/bring forward during the next meeting and leave a comment. I think it might be good to discuss something more general to push forward the development of DYNAMITE as we have not met in a while. We can leave git issues to another meeting unless they are very pressing.

Cheers,
Alina

**Comment by prashjet** [comment]
An idea: we could sketch out a plan for how to implement colouring.

(FYI - I'll be joining online on 14th)

**Reply by boeckera** [reply]
Great idea!
I guess we could make ourselves familiar again with the concept by reading this paper: https://ui.adsabs.harvard.edu/abs/2020MNRAS.496.1579Z/abstract

**Comment by boeckera** [comment]
Hello all @dynamics-of-stellar-systems/dynamite-maintainers,
don't forget this week's dynamite meeting! If you haven't done so already, please react in some sort of way, if you will join!
Cheers,
Alina

**Reply by prashjet** [reply]
Thomas and I are on zoom:

https://univienna.zoom.us/j/93990951862?pwd=T0wzNXlnWm1sN1BjelhScmd3WFJlZz09
